docker/README.md

```diff
@@ -30,11 +30,11 @@ Running MXNet Model Server with Docker in two steps:
 This will download the MMS Docker image and run its default configuration, serving a SqueezeNet model.
 
 ```bash
-docker run -itd --name mms -p 80:8080 -p 81:8081 awsdeeplearningteam/mms_cpu mxnet-model-server --start --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
+docker run -itd --name mms -p 80:8080 -p 8081:8081 awsdeeplearningteam/mms_cpu mxnet-model-server --start --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
```

Comment:
Is there a strong reason to do this? If yes, why are we leaving port 80? If there is no strong reason to do this, lets not do this change.

Reply:
default http port is 80, this is a well known port. It make sense to map to 80,
Most of the port number less then 1024 are well known and are reserved. 81 is not officially assigned, but usually associated with TorPark or Pinch.
We should avoid confusion it might be introduced.